app/controllers/incurred_incidentals_controller.rb

```diff
@@ -56,6 +57,20 @@ def update
 
   private
 
+  def upload_documents
+    # only do this on sucess and with a file
+    if @incurred_incidental.errors.empty? && params[:file]
+      # only allow types of uploaded file
+      params.require(:file).transform_values! { |uf| uf if uf.class == ActionDispatch::Http::UploadedFile }
+
+      params.require(:file).to_unsafe_h.map(&:itself).each do |id, uploaded_file|
```

Comment:
Line 64 strips out anything except anything but `ActionDispatch::Http::UploadedFile` types.
Line 66 iterates over the params, I make it an unsafe hash so I allow everything and so I have access to the map function.  (map is depricated in ActionParameters.)